packages/esm-ward-app/src/types/index.ts

```diff
@@ -44,7 +48,7 @@ export type WardPatient = {
 
 export interface WardPatientWorkspaceProps extends DefaultWorkspaceProps {
   wardPatient: WardPatient;
-  WardPatientHeader: React.FC<WardPatient>;
+  WardPatientHeader: WardPatientCardType;
```

Comment:
Why is the header a card type (instead of a header type)?    Or does "WardPatientCardType" a type for individual elements that can appear within a card?

is the "W" in WardPatientHeader meant to be capitalized?

Reply:
Both the elements and the overall card share the same type.

Yeah, the "W" is capitalized because it's a React component. It gets used as `<WardPatientHeader .../>`